### PR TITLE
check tfjs-vis is open before trainig

### PR DESCRIPTION
**Pull request by young31**
- after loading images, to push train button, someone might hide tfjs-vis window
- and they cannot know what's happening after push train button, 
so I think it is better to let them know what's goining on(training goes well)

### CODE REVIEW

**Comment by juandes** [comment]
Hi @young31, it is a valid suggestion. Before accepting, there are a few minor details I'd like to ask. First, could you add a short comment above the statement explaining the reason of the change? Also, please make sure you are not changing the document's format because I'd like to keep the consistency with the other files. So the only change should be the your statement and the document. Also, please take a look at the contribution guide and name the branch accordingly. You can find it here: https://github.com/Apress/Practical-TensorFlow.js/blob/master/Contributing.md.

Again, thanks for the PR!

File: 4/index.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ function defineModel() {
 }
 
 async function train() {
+  tfvis.visor().open();
```
Comment (juandes) [inline 4/index.js:76]: Please add a comment explaining the change.

File: 4/index.js [hunk 2]
```diff
@@ -100,10 +101,7 @@ async function train() {
     epochs: 30,
     validationData: [xTest, yTest],
     shuffle: true,
-    callbacks: tfvis.show.fitCallbacks(
-      { name: 'Loss and Accuracy', tab: 'Training' },
-      ['loss', 'val_loss', 'acc', 'val_acc'],
-    ),
+    callbacks: tfvis.show.fitCallbacks({ name: 'Loss and Accuracy', tab: 'Training' }, ['loss', 'val_loss', 'acc', 'val_acc'], ),
```
Comment (juandes) [inline 4/index.js:104]: Avoid changing the format.

File: 4/index.js [hunk 3]
```diff
@@ -222,15 +220,15 @@ function enableButton(selector) {
 function init() {
   prepareCanvas();
   createButton('Load data', '#pipeline', 'load-btn',
-    async () => {
+    async() => {
```
Comment (juandes) [inline 4/index.js:223]: Avoid changing the format.

File: 4/index.js [hunk 4]
```diff
@@ -222,15 +220,15 @@ function enableButton(selector) {
...
       data = new MnistData();
       await data.load();
       drawData();
       enableButton('train-btn');
     });
 
   createButton('Train', '#pipeline', 'train-btn',
-    async () => {
+    async() => {
```
Comment (juandes) [inline 4/index.js:231]: Avoid changing the format.

**Comment by young31** [comment]
First, thanks for replying. I didn't realize my formatter changes something. Before I commit, I want to confirm everything is in the right ways. 
I will add a comment like below  

async function train() {
  // Make sure the tfjs-vis visor is open.
  tfvis.visor().open();
  
And I'm not sure in 'git checkout -b my_code_contribution', how can I name my_code_contribution right way. Just like make_visor_open is OK? 
  
Thanks for reviewing and replying again.

**Comment by juandes** [comment]
> First, thanks for replying. I didn't realize my formatter changes something. Before I commit, I want to confirm everything is in the right ways.
> I will add a comment like below
> 
> async function train() {
> // Make sure the tfjs-vis visor is open.
> tfvis.visor().open();
> 
> And I'm not sure in 'git checkout -b my_code_contribution', how can I name my_code_contribution right way. Just like make_visor_open is OK?
> 
> Thanks for reviewing and replying again.

Hi @young31, the comment is great! I'd modify it a bit to explain the why, for example, "Make sure the tfjs-vis visor is open before starting the training." As for the branch name, yours "make_visor_open" is good :). Thanks for the comment! I really appreciate the contribution :).

**Pull request closed without merging**